pkg/config/mesh/mesh.go

```diff
@@ -91,7 +95,18 @@ func DefaultMeshConfig() meshconfig.MeshConfig {
 		EnableAutoMtls:                    &types.BoolValue{Value: false},
 		ThriftConfig:                      &meshconfig.MeshConfig_ThriftConfig{},
 		LocalityLbSetting:                 &v1alpha3.LocalityLoadBalancerSetting{},
+		ClusterLocalNamespaces:            append(make([]string, 0), defaultClusterLocalNamespaces...),
```

Comment:
nit: no need append

Reply:
there is actually ... each instance needs to have its own array. You don't want changes to one instance modifying the default array.

Reply:
What instance do you mean here?